packages/cmrjs/tests/fixtures/MOD09GQ.A3411593.1itJ_e.006.9747594822314.cmr.json

```diff
@@ -0,0 +1 @@
+{"SpatialExtent":{"HorizontalSpatialDomain":{"Geometry":{"BoundingRectangles":[{"WestBoundingCoordinate":-180,"EastBoundingCoordinate":180,"NorthBoundingCoordinate":90,"SouthBoundingCoordinate":-90}]}}},"ProviderDates":[{"Date":"2018-12-19T17:30:31.424Z","Type":"Insert"}],"DataGranule":{"DayNightFlag":"Unspecified","ProductionDateTime":"2016-01-09T11:40:45.032Z","ArchiveAndDistributionInformation":[{"Name":"Not provided","Size":1.009857177734375,"SizeUnit":"NA"}]},"TemporalExtent":{"RangeDateTime":{"BeginningDateTime":"2016-01-09T11:40:45.032Z","EndingDateTime":"2016-01-09T11:41:12.027Z"}},"GranuleUR":"MOD09GQ.A3411593.1itJ_e.006.9747594822314","CollectionReference":{"ShortName":"MOD09GQ","Version":"006"},"RelatedUrls":[{"URL":"https://nasa.github.io/cumulus/docs/cumulus-docs-readme","Type":"GET DATA"},{"URL":"http://localhost:5002/cumulus-test-sandbox-protected/MOD09GQ___006/2016/MOD/MOD09GQ.A3411593.1itJ_e.006.9747594822314.hdf","Description":"File to download","Type":"GET DATA"},{"URL":"https://cumulus-test-sandbox-public.s3.amazonaws.com/MOD09GQ___006/MOD/MOD09GQ.A3411593.1itJ_e.006.9747594822314_ndvi.jpg","Description":"File to download","Type":"GET RELATED VISUALIZATION"},{"URL":"http://localhost:5002/cumulus-test-sandbox-protected-2/MOD09GQ___006/MOD/MOD09GQ.A3411593.1itJ_e.006.9747594822314.cmr.json","Description":"File to download","Type":"EXTENDED METADATA"},{"URL":"http://localhost:5002/s3credentials","Description":"api endpoint to retrieve temporary credentials valid for same-region direct s3 access","Type":"VIEW RELATED INFORMATION"}],"MetadataSpecification":{"URL":"https://cdn.earthdata.nasa.gov/umm/granule/v1.5","Name":"UMM-G","Version":"1.5"}}
```

Comment:
I'm uncertain if these fixtures should go in the testData package, prior to merging I'll move them over if appropriate, just didn't want it to impede review.

Reply:
I'm not really sure when something needs to be in test-data package. good question.

Reply:
It seems in practice that things end up in `test-data` when they are used in tests across multiple packages, as a single package can simply contain its own fixtures (test-data shouldn't be polluted with one-off fixtures, imho). I'd advocate that's the best criterion, is it used across packages?

Reply:
@mennovandiermen  that's fair.    Perhaps the better question is, should any of these be using fixtures *in* that package.